membershiporganization/migrations/0008_rename_realend_to_openended.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2021-02-02 19:54
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+def map_new_type(apps, schema_editor):
```

Comment:
I copied this migration approach from the corresponding migration changing the open-ended field from a Boolean to a string field accepting the controlled vocabulary [in the `composition` app](https://github.com/security-force-monitor/sfm-cms/blob/master/composition/migrations/0006_auto_20170905_2006.py).